### DISCUSSION TITLE
part-ii/2019-115/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#49] [Question comments]
Part II 2019 Principles of Statistics: Paper 2, Section II, J 

https://questions.tripos.org/part-ii/2019-115/

**Comment by ArthurConmy** [comment]
I get $\pi_{MLE} = \text{min} \left( 1, \frac{n-z}{n(1-e^{-\lambda} )} \right)$ where $z$ is the number of zeroes in the sample $Y_1, ... , Y_n$. Then this leads to a test statistic asymptotically distributed as $\text{min}(0, Y)$, where

$$Y \sim N(0, \frac{e^{-\lambda }}{1 - e^{-\lambda }})$$

**Reply by AustenLamacraft** [reply]
I just edited this (hopefully correctly) because there was an uncaught $\LaTeX$ error that caused tripos-comments to barf. Will have to check that out.
